Cargo.toml

```diff
@@ -401,4 +401,5 @@ zip-extract = "=0.2.1"
 # revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors.git", rev = "a625c04" }
 
 ## foundry
+foundry-compilers = { git = "https://github.com/foundry-rs/compilers.git", rev = "10fe5af" }
```

Comment:
Pending release of https://github.com/foundry-rs/compilers, cc @mattsse when time permits would be great if you could publish a new version

Reply:
done